api_client/python/timesketch_api_client/config.py

```diff
@@ -152,21 +57,41 @@ class ConfigAssistant:
         'client_secret',
     ])
 
+    CONFIG_ORDERING = {
+        'host_uri': 1,
+        'auth_mode': 2,
+        'username': 3,
+        'client_id': 4,
+        'client_secret': 5,
+        'password': 6,
+    }
+
+    CONFIG_HINTS = {
+        'host_uri': 'URL of the Timesketch server',
+        'username': 'The username of the Timesketch user',
+        'password': 'Password of the chosen user.',
+        'auth_mode': (
+            'Authentication mode, valid choices are: "timesketch" '
```

Comment:
I would opt to call it 'userpass' instead of 'timesketch' as it is a bit more clear on what it is.

Reply:
ok, agree.. changed it everywhere, since it needs to be changed in several places